### PR TITLE
Various hotfixes for v1.3

### PR DESCRIPTION
**Pull request by AaronCQL**

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonalBestTable.java [hunk 1]
```diff
@@ -41,6 +41,7 @@ public PersonalBestTable(ObservableList<Exercise> e) {
         exerciseDate.setCellValueFactory(new PropertyValueFactory<Exercise, String>("exerciseDate"));
         exerciseWeight.setCellValueFactory(new PropertyValueFactory<Exercise, String>("exerciseWeight"));
         exerciseReps.setCellValueFactory(new PropertyValueFactory<Exercise, String>("exerciseReps"));
+        exerciseName.setText("Personal Best Exercise");
```
Comment (Yonggiee) [inline src/main/java/seedu/address/ui/PersonalBestTable.java]: I suggest moving this to fxml for consistency

File: src/main/resources/view/PersonalBestTable.fxml [hunk 2]
```diff
@@ -5,8 +5,8 @@
 
 <TableView fx:id="tableView" prefHeight="200.0" prefWidth="250.0" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
     <columns>
+        <TableColumn fx:id="exerciseName" editable="false" minWidth="200.0" prefWidth="200.0" reorderable="false" text="Exercise" />
```
Comment (Yonggiee) [inline src/main/resources/view/PersonalBestTable.fxml]: suggest text="Personal Best Exercise"

File: src/main/java/seedu/address/ui/PersonalBestTable.java (hunk 1 above)
Comment (AaronCQL) [reply src/main/java/seedu/address/ui/PersonalBestTable.java]: Ok, please kindly do it for me  =D

Comment (Yonggiee) [reply src/main/java/seedu/address/ui/PersonalBestTable.java]: damn

**Review by Yonggiee — APPROVED** [review]
LGTM

**Pull request merged**